frontend-opal/.eslintrc.json

```diff
@@ -20,13 +21,21 @@
     },
     "plugins": [
         "react",
-	    "react-native"
+        "react-native"
     ],
     "parser": "babel-eslint",
     "rules": {
-	    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+        "react/jsx-filename-extension": [
```

Comment:
@hondahmd this file is still using 4 spaces for indentation, is this on purpose?